utils/types.go

```diff
@@ -24,6 +24,24 @@ type Kubernetes struct {
 	NodeName   string `json:"node_name"`
 }
 
+type Args struct {
+	Mesos Mesos `json:"org.apache.mesos,omitempty"`
+}
+
+type Mesos struct {
+	NetworkInfo NetworkInfo `json:"network_info"`
+}
+
+type NetworkInfo struct {
+	Name string `json:"name"`
+	Labels struct {
```

Comment:
I can understand pulling them apart for developer readability, but I wrote them this way because there's zero re usability (and likely won't be any in the future) for those two structs. I like the idea of bundling NetworkInfo into one single object